<ui version="4.0" >
 <class>UIXUPManager</class>
 <widget class="QDockWidget" name="UIXUPManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Projects Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QFrame" name="fToolButtons" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QToolBar" name="tbActions" >
         <property name="font" >
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="iconSize" >
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="floatable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="sView" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QStackedWidget" name="swViews" >
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <widget class="QTreeView" name="tvProjects" >
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
       <widget class="QTreeView" name="tvProxiedProjects" >
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </widget>
      <widget class="QTextEdit" name="teLog" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
